.buildkite/commands/checkout-release-branch.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -eu
 
-# RELEASE_VERSION is passed as an environment variable from fastlane to Buildkite
-#
+# RELEASE_VERSION is passed as an environment variable passed to Buildkite by ReleasesV2.
 if [[ -z "${RELEASE_VERSION}" ]]; then
```

Comment:
> It would also make the implementation consistent with the recent changes accessing RELEASE_VERSION in the pipeline command that we made in some iOS projects.

I think this is a good idea. It's an area where we lack some standardization.
Besides, having these scripts relying less on environment variables and having those explicit in the pipelines make things clearer and easier to follow.